components/signersForm.tsx

```diff
@@ -100,12 +100,14 @@ const SignersForm: FC<{
   const duration = useMemo(() => {
     if (
       ["0.0.6", "0.0.8", "0.0.9", "unknown version"].includes(
```

Comment:
the latest version is `0.1.1`. should we add it? is it possible that we do something here so when the next time adding the new version, we won't forget here?

Reply:
We don't need to, this filter is here because old contracts didn't have the `effective_period`